modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -993,6 +999,67 @@ public CodegenParameter fromRequestBody(RequestBody body, Set<String> imports, S
         return codegenParameter;
     }
 
+    @Override
+    public String getTypeDeclaration(String name) {
+        return "models::" + name;
+    }
+
+    private String modelFromSchema(Schema schema) {
+        String ref = null;
+
+        if (schema != null) {
+            ref = schema.get$ref();
+        }
+
+        if (ref != null && ref.indexOf("#/components/schemas/") == 0) {
+            ref = toModelName(ref.substring("#/components/schemas/".length()));
+        } else {
+            ref = null;
+        }
+
+        return ref;
+    }
+
+    @Override
+    public String getTypeDeclaration(Schema p) {
+        LOGGER.info("Getting type declaration for schema");
```

Comment:
should we change these LOGGER.info to LOGGER.debug instead?

Reply:
Ah yes - possibly trace actually.